mappings/net/minecraft/block/BlockState.mapping

```diff
@@ -200,3 +201,4 @@ CLASS net/minecraft/class_2680 net/minecraft/block/BlockState
 		ARG 2 state
 		ARG 3 hitResult
 		ARG 4 projectile
+	METHOD method_20827 isSolidFullSquare (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;)Z
```

Comment:
How can a 3D model be a square? Or is this side-specific, in which case the method should probably say so?

Reply:
It's for checking if a specific side of a blockstate is a solid full square. It uses the same name as the method it calls from `Block`.

Reply:
isSideSolidFullSquare, then

Reply:
Well then should the method in `Block` not also be `isSideSolidFullSquare`?

Reply:
Correct

Reply:
![image](https://user-images.githubusercontent.com/2484721/61578801-6fe3cc00-ab3f-11e9-8485-ea4717e99230.png)
This is the code, the method in `Block` is also side specific and is named `isSolidFullSquare`. I named the `BlockState` method the same for consistency. I can change both, but I don't see why one should be different to the other.

Reply:
Done